packages/manager/modules/nutanix/src/dashboard/general-info/controller.js

```diff
@@ -38,6 +36,14 @@ export default class NutanixGeneralInfoCtrl {
     this.clusterRedeploying = this.cluster.status === CLUSTER_STATUS.DEPLOYING;
     this.showRedeployWarningModal = false;
     this.TRACKING = TRACKING;
+
+    const { ovhSubsidiary } = this.user;
+    this.NUTANIX_LINK =
+      this.packType === this.NUTANIX_PERSONAL_LICENSE_EDITION
+        ? GUIDE_PACKAGES_URL.BYOL[ovhSubsidiary] ||
+          GUIDE_PACKAGES_URL.BYOL.DEFAULT
+        : GUIDE_PACKAGES_URL.DEFAULT[ovhSubsidiary] ||
+          GUIDE_PACKAGES_URL.DEFAULT.DEFAULT;
   }
```

Comment:
Can we remove the the suffix in constant file and handle it on controller file ? To avoid duplicate constants
```suggestion
    this.NUTANIX_LINK =
      this.packType === this.NUTANIX_PERSONAL_LICENSE_EDITION
        ? (GUIDE_PACKAGES_URL[ovhSubsidiary] ||
      GUIDE_PACKAGES_URL.DEFAULT)  + 'byol'
        : (GUIDE_PACKAGES_URL.DEFAULT[ovhSubsidiary] ||
          GUIDE_PACKAGES_URL.DEFAULT.DEFAULT) + 'packed' ;
  }
```